packages/fxa-auth-client/lib/recoveryKey.ts

```diff
@@ -0,0 +1,42 @@
+import { jweEncrypt, hkdf } from './crypto';
+import { hexToUint8, uint8ToHex } from './utils';
+
+export async function generateRecoveryKey(
+  uid: string,
+  keys: { kA: string; kB: string }
```

Comment:
Same type as https://github.com/mozilla/fxa/pull/6671/files#diff-4b170feaa918643ed9b30065a372669dc87993d5e83113cf851077783c46f6beR454-R457 ?  If so, maybe define a type for it.

Reply:
yeah im half-way through changing the types. I kind of wonder if labeling `hexstring` is worth it or not since it's just an alias. 🤔 ... thoughts?

Reply:
I like it because then the readers know that they are hex strings.